project/AppDependencies.scala

```diff
@@ -4,36 +4,34 @@ import sbt._
 
 object AppDependencies {
 
-  val hmrc = "uk.gov.hmrc"
-
   val compile = Seq(
     ws,
-    hmrc                %% "domain"                     % "5.11.0-play-26",
-    hmrc                %% "mongo-lock"                 % "7.0.0-play-26",
-    hmrc                %% "bootstrap-backend-play-26"  % "5.3.0",
-    hmrc                %% "simple-reactivemongo"       % "8.0.0-play-26",
+    "uk.gov.hmrc"       %% "domain"                     % "6.2.0-play-28",
+    "uk.gov.hmrc"       %% "mongo-lock"                 % "7.0.0-play-28",
+    "uk.gov.hmrc"       %% "bootstrap-backend-play-28"  % "5.10.0",
+    "uk.gov.hmrc"       %% "simple-reactivemongo"       % "8.0.0-play-28",
     "com.eclipsesource" %% "play-json-schema-validator" % "0.9.4",
-    "org.typelevel"     %% "cats-core"                  % "2.0.0",
-    "com.github.kxbmap" %% "configs"                    % "0.6.0",
+    "org.typelevel"     %% "cats-core"                  % "2.5.0",
+    "com.github.kxbmap" %% "configs"                     % "0.6.0",
     compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full),
     "com.github.ghik" % "silencer-lib" % "1.7.1" % Provided cross CrossVersion.full
   )
 
   val test = Seq(
-    hmrc                   %% "service-integration-test"    % "1.1.0-play-26"    % "test",
-    hmrc                   %% "stub-data-generator"         % "0.5.3"             % "test",
+    "uk.gov.hmrc"          %% "service-integration-test"    % "1.1.0-play-28"     % "test",
```

Comment:
i doubt if we are really using this dep `service-integration-test`. , please double check and remove if not needed.